src/pages/deployment/custom-domains.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Custom Domains
+---
+
+Custom domains can be configures inside of the Deployment -> Domains tab
+
+Press edit, type in your domain, and we'll prompt you to update your CNAME
+
+<NextImage  src="/images/domain.png" 
+            alt="Screenshot of Custom Domain"
+            layout="responsive"
+            width={1205} 
+            height={901}
+            quality={100} />
+
+Once the domain is set, head over to your favorite registrar and add the DNS record as directed
+
+NOTE! Sometimes this takes a while due to DNS
+
+## Provider Specific Instructions
+
+Enabling OrangeCloud Proxy
+
+If you're on CloudFlare, you MUST set your DNS settings to full or above. Otherwise, Railway will fail to perform a secure handshake and proxy your DNS
```

Comment:
add a description about turning off cf proxy for specific sub-domains, in the case of someone not wanting to make their entire domain use full or above.

Reply:
heres an image of what I'm talking about: ![image](https://user-images.githubusercontent.com/35403473/116495989-e6574700-a858-11eb-87ed-6b5a0844dcf5.png)